lib/fastlane/actions/oclint.rb

```diff
@@ -40,9 +65,9 @@ def self.run(params)
           '"' + files.join('" "') + '"'
         ].join(' ')
 
-        Action.sh command
-
         Actions.lane_context[SharedValues::FL_OCLINT_REPORT_PATH] = File.expand_path(report_path)
+
+        Action.sh command # Needs to be last as return value for tests
```

Comment:
You could remove the comment and just add a `return` statement to make it clear. Could you also add `(` `)` around the `command`?